### DISCUSSION TITLE
OGCS not working with Outlook iOS app

### DISCUSSION THREAD

**Discussion by phw198** [phw198/OutlookGoogleCalendarSync#1892] [General]
I have been using Outlook Google calendar sync for a while now, I cannot get it for some reason to work with my Outlook IOS app. Am I doing something wrong? .
Otherwise works great with my Google, and regular Outlook calendars .
Thanks in advance.

_Originally posted by @heshie75v in https://github.com/phw198/OutlookGoogleCalendarSync/discussions/1882#discussioncomment-9896389_

**Comment by phw198** [comment]
@heshie75v Can you provide some more information about the Outlook iOS app? Do you mean you are syncing to an Outlook calendar with OGCS, but when you configure Outlook on iOS to the same account, you can't see the changes that OGCS is making?

**Comment by TheJacksonian** [comment]
I'm no expert, but the iOS implementation of calendars seems a little odd, I find it does the 'sync' per account, @heshie75v if you check your mail in iOS first does the calendar update?

**Comment by heshie75v** [comment]
Calendar and mail work perfectly on my Iphone IOS version of outlook. That’s what I use for my mail,

**Comment by heshie75v** [comment]
Outlook IOS does not sync with OGCS, however my OGCS works perfectly otherwise e.g. outlook for my desktop and google works just fine, IOS and desktop.

**Comment by phw198** [comment]
@heshie75v You're not really explaining what the problem is, but "Outlook IOS does not sync with OGCS" is correct: OGCS syncs with your Microsoft account calendar, and iOS also syncs with the same account, so should pick up any changes OGCS makes.

If you check in Outlook Online web view (https://outlook.live.com/calendar/0/view/week) and the changes from OGCS show there, but not on your iOS Outlook app, then the problem is not with OGCS, but iOS.

If the changes are not in Outlook Online, but are in the Outlook client on the same machine OGCS is installed on, then the issue is with the Outlook client syncing changes from your local computer.
